Cannot find module 'vue-template-compiler/build'

**Describe the bug**
after `yarn global add @vuese/cli`, use the cammand `vuese` or `vuese --version` or `vuese gen` and so on.
it cannot work and the error message like this:

internal/modules/cjs/loader.js:1032
  throw err;
  ^

Error: Cannot find module 'vue-template-compiler/build'
Require stack:
- /Users/shiwj/.nvm/versions/node/v14.4.0/lib/node_modules/@vuese/cli/node_modules/@vuese/parser/dist/index.js
- /Users/shiwj/.nvm/versions/node/v14.4.0/lib/node_modules/@vuese/cli/dist/index.js
- /Users/shiwj/.nvm/versions/node/v14.4.0/lib/node_modules/@vuese/cli/bin/index.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:1029:15)
    at Function.Module._load (internal/modules/cjs/loader.js:898:27)
    at Module.require (internal/modules/cjs/loader.js:1089:19)
    at require (internal/modules/cjs/helpers.js:73:18)
    at Object.<anonymous> (/Users/shiwj/.nvm/versions/node/v14.4.0/lib/node_modules/@vuese/cli/node_modules/@vuese/parser/dist/index.js:7:13)
    at Module._compile (internal/modules/cjs/loader.js:1200:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1220:10)
    at Module.load (internal/modules/cjs/loader.js:1049:32)
    at Function.Module._load (internal/modules/cjs/loader.js:937:14)
    at Module.require (internal/modules/cjs/loader.js:1089:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/shiwj/.nvm/versions/node/v14.4.0/lib/node_modules/@vuese/cli/node_modules/@vuese/parser/dist/index.js',
    '/Users/shiwj/.nvm/versions/node/v14.4.0/lib/node_modules/@vuese/cli/dist/index.js',
    '/Users/shiwj/.nvm/versions/node/v14.4.0/lib/node_modules/@vuese/cli/bin/index.js'
  ]
}


**Environmental information:**
@vuese/cli@ v2.13.1
node v14.4.0
npm v6.14.5

--- a/packages/parser/package.json
+++ b/packages/parser/package.json
@@ -15,6 +15,13 @@
   "files": [
     "dist"
   ],
+  "dependencies": {
+    "@babel/generator": "^7.10.3",
+    "@babel/parser": "^7.10.3",
+    "@babel/traverse": "^7.10.3",
+    "@babel/types": "^7.3.0",
+    "vue-template-compiler": "^2.6.11"
+  },
   "scripts": {
     "tsc": "rm -rf dist && tsc"
   },

